tests/foreman/cli/test_organization.py

```diff
@@ -1693,3 +1694,27 @@ def test_positive_info_by_name(self):
         org = make_org()
         result = Org.info({'name': org['name']})
         self.assertEqual(org['id'], result['id'])
+
+    @tier1
+    def test_positive_multibyte_latin1_org_names(self):
+        """Hammer Multibyte and Latin-1 Org names break list pagination
+
+        :id: 4fa0afe7-6d0a-4c3e-a0fc-4ecb95c50fc9
+
+        :BZ: 1418412
+
+        :expectedresults: Multibyte and latin1 names need to be
+            displayed with consistent spacing
+        """
+        org_names = valid_org_names_list()
+        for org in org_names:
+            make_org({'name': org})
+        org_list = [line for line in Org.list(output_format='table') if line]
```

Comment:
What if Org.list() hasn't returned our latin1 names at all (because of a bug or pagination)? Imo it's worth asserting `org_names` elements are present inside strings of `org_list`.